### PR TITLE
chore: skip local repos on pre-commit.ci

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+ci:
+  skip: [mypy-code, localhost-examples]
```
Comment (RobPasMue) [inline .pre-commit-config.yaml:2]:
```suggestion
# ci:
#   skip: [mypy-code, localhost-examples]

```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# ci:
+#   skip: [mypy-code, localhost-examples]
+
+
 repos:
```
Comment (RobPasMue) [inline .pre-commit-config.yaml:5]:
```suggestion
ci:
  skip: [mypy-code, localhost-examples]

repos:

```

**Comment by RobPasMue** [comment]
Requested in https://github.com/ansys/grantami-jobqueue/pull/36#discussion_r1540008689 by @Andy-Grigg

**Review by ludovicsteinbach — APPROVED** [review]
I wasn't even aware this was a thing !
Thanks Roberto

**Comment by RobPasMue** [comment]
Anytime - feel free to merge!

**Pull request merged**